studio/components/interfaces/Database/Indexes/Indexes.constants.ts

```diff
@@ -0,0 +1,32 @@
+export const INDEX_TYPES = [
+  {
+    name: 'B-Tree',
+    value: 'btree',
+    description: 'For querying data with equality or range conditions on columns.',
+  },
```

Comment:
Worth mentioning that B-Tree is the default / recommended index type for users who are unsure of what to pick?

Reply:
yup good idea - just pushed!